deploy/charts/alluxio/templates/csi/csi-fuse.yaml

```diff
@@ -118,7 +118,7 @@ data:
       volumes:
         - name: alluxio-fuse-mount
           hostPath:
-            path: {{ $hostMountPath }}
+            path: {{ .Values.fuse.hostPathForMount | dir }}
```

Comment:
do we need a default value for this?

Reply:
.Values.fuse.hostPathForMount is default to `/mnt/alluxio/fuse`